client/tests/sdk.test.ts

```diff
@@ -90,5 +90,33 @@ describe('SDK', () => {
         verifyContractBytecode(sdk, contract.bytecode, wrongSource)
       ).rejects.toThrow();
     });
-  }, 10000);
+  }, 20000);
```

Comment:
This is now the closing of `describe` which does not accept a timeout. From my testing the timeout (even in the correct place) is no longer necessary.
```suggestion
  });
```